test/test_helper.rb

```diff
@@ -167,6 +168,7 @@ class ActionController::TestCase
 
   setup do
     request.env['warden'] = stub(authenticate!: false, authenticated?: false, user: nil)
+    stub_content_register_policies
```

Comment:
Ideally, there would be a test double for the content register that is used in test mode, but I think writing one of those is beyond the scope of this work and should be done as part of the work to extract a separate gem for the content register API.